adafruit_thermal_printer/thermal_printer.py

```diff
@@ -117,13 +123,17 @@ class _PrintModeBit:
         # which by design only implements get, set, init.  As a result workaround
         # this pylint issue by disabling the warning.
         # pylint: disable=too-few-public-methods
-        def __init__(self, mask):
+        def __init__(self, mask: int) -> None:
             self._mask = mask
 
-        def __get__(self, obj, objtype):
+        def __get__(
+            self,
+            obj: Optional["ThermalPrinter"],
+            objtype: Optional[Type["ThermalPrinter"]],
+        ) -> bool:
             return obj._print_mode & self._mask > 0
 
-        def __set__(self, obj, val):
+        def __set__(self, obj: Optional["ThermalPrinter"], val: int) -> None:
```

Comment:
`__set__()` can only be called on an instance, so `obj` can never evaluate to `None`, so it can just be `"ThermalPrinter"`.